pkg/api/apikey.go

```diff
@@ -9,6 +9,28 @@ import (
 	"github.com/grafana/grafana/pkg/models"
 )
 
+func GetAPIKeyCurrent(c *models.ReqContext) Response {
+	query := models.GetApiKeyByIdQuery{ApiKeyId: c.ApiKeyId}
+
+	if err := bus.Dispatch(&query); err != nil {
+		return Error(500, "Failed to list api keys", err)
+	}
+
+	var expiration *time.Time = nil
+	if query.Result.Expires != nil {
+		v := time.Unix(*query.Result.Expires, 0)
+		expiration = &v
```

Comment:
Can this be done in one line? :)

Reply:
@askomorokhov it's a nit ))

Reply:
Unfortunately no, time.Unix is a function

Reply:
```go
expiration = pointer.ToTime(time.Unix(*query.Result.Expires, 0))
```

https://pkg.go.dev/github.com/AlekSi/pointer#ToTime

Reply:
😊 code won't be consistent :)
Then I'll have to replace it everywhere

Reply:
Probably. Just pointing out that it _can_ be; not necessary _should_ be.